src/config/configDefaults.ts

```diff
@@ -0,0 +1,8 @@
+export const SHOW_MUTATION_ASSESSOR = true;
```

Comment:
@leexgh This works, but it might be better to have a single source of truth (i.e. one configurable variable) for the concept of whether to include mutation assessor? E.g. this could also be:

```
export const SHOW_MUTATION_ASSESSOR = ANNOTATION_QUERY_FIELDS.includes('mutation_assessor')
```
In some ways it might be cleaner to at some point move this elsewhere, since it's not really user/developer config but rather a variable derived from config

We can leave it like as is for now, but some food for thought for when we have more configuration